dogs/src/main/res/raw/dog.json

```diff
@@ -1 +1,4212 @@
\ No newline at end of file
+{
```

Comment:
Can we have it ignore json files? This actually increases file size and time to parse.

Reply:
this wasn't detekt, this was intellij formatting applied to the whole project, so it won't normally run

Reply:
@elihart Can you revert this one?